### PR TITLE
[SWDEV-342188] [MSRCHA-236][W/A] disable group conv in asm igemm solver

### PR DESCRIPTION
**Pull request by shaojiewang**
disable group conv for rocm5.2 staging

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -1332,7 +1332,7 @@ COMMAND ${DYNAMIC_IMPLICITGEMM_XDLOPS_NHWC_FWD_ENVS} $<TARGET_FILE:test_conv2d>
...
 COMMAND ${DYNAMIC_IMPLICITGEMM_XDLOPS_NHWC_FWD_ENVS} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --verbose --input  16   3  17  17 --weights  64   3 1 1 --pads_strides_dilations 0 0 1 1 1 1 --disable-backward-data --disable-backward-weights
 COMMAND ${DYNAMIC_IMPLICITGEMM_XDLOPS_NHWC_FWD_ENVS} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --verbose --input   2  64  19  19 --weights 510  64 3 3 --pads_strides_dilations 1 1 1 1 1 1 --disable-backward-data --disable-backward-weights
-COMMAND ${DYNAMIC_IMPLICITGEMM_XDLOPS_NHWC_FWD_ENVS} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --verbose --input  16   3 224 224 --weights  63   1 3 3 --pads_strides_dilations 1 1 1 1 1 1 --group-count 3 --disable-backward-data --disable-backward-weights
+#COMMAND ${DYNAMIC_IMPLICITGEMM_XDLOPS_NHWC_FWD_ENVS} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --verbose --input  16   3 224 224 --weights  63   1 3 3 --pads_strides_dilations 1 1 1 1 1 1 --group-count 3 --disable-backward-data --disable-backward-weights
```
Comment (junliume) [inline test/CMakeLists.txt:1335]: Why we disable this test?

**Comment by junliume** [comment]
> disable group conv for rocm5.2 staging

You can change the base branch from webpage `Edit`,  no need to close and start another PR :-)

**Comment by junliume** [comment]
@shaojiewang Thanks and please keep us updated about the test run results on gfx908 and gfx90a :)

**Comment by shaojiewang** [comment]
> > disable group conv for rocm5.2 staging
> 
> You can change the base branch from webpage `Edit`, no need to close and start another PR :-)

I see. Thanks for reminding.

**Comment by shaojiewang** [comment]
> @shaojiewang Thanks and please keep us updated about the test run results on gfx908 and gfx90a :)

Sure, the gfx90a's process looks like finishing. I sent the log via teams. gfx908's process is still running in about 30 epoch.

**Review by carlushuang — APPROVED** [review]

**Review by junliume — APPROVED** [review]

**Pull request merged**